_posts/2023-08-23-even-friendlier-sql.md

```diff
@@ -0,0 +1,596 @@
+│ The Motion Picture                     │ named          │ The Motion Picture │     NULL │
+│ 2                                      │ numbered       │ NULL               │        2 │
+│ 3                                      │ numbered       │ NULL               │        3 │
+│ 4                                      │ numbered       │ NULL               │        4 │
+│ 5                                      │ numbered       │ NULL               │        5 │
+│ 6                                      │ numbered       │ NULL               │        6 │
+│ First Contact                          │ named          │ First Contact      │     NULL │
+└────────────────────────────────────────┴────────────────┴────────────────────┴──────────┘
+```
+
+## Additional friendly features
+
+Several other friendly features are worth mentioning and some are powerful enough to warrant their own blog posts. 
+
+DuckDB takes a nod from the [`describe` function in Pandas](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.describe.html) and implements a `SUMMARIZE` keyword that will calculate a variety of statistics about each column in a dataset for a quick, high-level overview. Simply prepend `SUMMARIZE` to any table or `SELECT` statement. 
+
+Have a look at the [correlated subqueries post](https://duckdb.org/2023/05/26/correlated-subqueries-in-sql.html) to see how to use subqueries that refer to each others’ columns. DuckDB’s advanced optimizer completely solves performance problems when expressing a query as naturally as possible. What was once an anti-pattern for performance reasons can now be used freely!
+
+DuckDB has added more ways to `JOIN` tables together that make expressing common calculations much easier. Some like `LATERAL`, `ASOF`, `SEMI`, and `ANTI` joins are present in other systems, but have high performance implementations in DuckDB. DuckDB also adds a new `POSITIONAL` join that combines by the row numbers in each table to match the commonly used Pandas capability of joining on row number indexes. See the [`JOIN` documentation](https://duckdb.org/docs/sql/query_syntax/from.html) for details, and look out for a blog post describing DuckDB’s state of the art `ASOF` joins!
+
+## Summary and Future Work
+
+DuckDB aims to be the easiest database to use. Fundamental architectural decisions to be in-process, have zero dependencies, and have strong typing contribute to this goal, but the friendliness of its SQL dialect has a strong impact as well. By extending the industry-standard PostgreSQL dialect, DuckDB aims to provide the simplest way to express the data transformations you need. These changes range from altering the ancient clause order of the `SELECT` statement to begin with `FROM`, allowing a fundamentally new way to use functions with chaining, to advanced nested data type calculations like list comprehensions.
+
+Future work for friendlier SQL includes:
```

Comment:
I would add, since work is already quite advanced, a generic "improved UX around extensions" (-> extension autoloading) and something about globbing files maybe?

Reply:
I took a swag at this - let me know if I got it right!